Website/src/rendering/src/components/Speakers/FeaturedSpeakers.tsx

```diff
@@ -1,50 +1,55 @@
-import Link from 'next/link';
-import { Text, Image } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Text, Field, Image } from '@sitecore-jss/sitecore-jss-nextjs';
 import { ComponentProps } from 'lib/component-props';
 import { GraphQLSpeaker } from 'src/types/speaker';
+import Link from 'next/link';
 
 export type FeaturedSpeakersProps = ComponentProps & {
   fields: {
     data: {
+      source: {
+        numberOfSpeakers: Field<string>;
+      };
       item: {
         children: {
           results: GraphQLSpeaker[];
         };
       };
     };
   };
-  params: {
-    NumberOfSpeakers: string;
-  };
 };
 
 const FeaturedSpeakers = (props: FeaturedSpeakersProps): JSX.Element => {
-  const speakers =
-    props.fields.data?.item?.children?.results &&
-    props.fields.data.item.children.results
-      .filter((item) => item.featured.value)
-      .sort()
-      .slice(0, parseInt(props.params.NumberOfSpeakers))
-      .map((speaker, index) => (
-        <Link key={index} href={`/speakers/${speaker.itemName}`} passHref>
-          <a className="grid-item">
-            <Image
-              field={speaker.picture.jsonValue}
-              alt={speaker.name}
-              width={265}
-              height={265}
-              loading="lazy"
-            />
+  const speakers = props.fields.data.item.children.results
+    .filter((item) => item.featured.value)
+    .sort()
+    .slice(0, parseInt(props.fields.data.source?.numberOfSpeakers?.value))
+    .map((speaker, index) => (
+      <Link key={index} href={`/speakers/${speaker.itemName}`}>
+        <a>
+          <div className="grid-item">
+            <div className="item-image">
+              <Image
+                field={speaker.picture.jsonValue}
+                alt={speaker.name.value}
+                width={265}
+                height={265}
+                loading="lazy"
+              />
+            </div>
             <div className="item-details">
-              <Text tag="p" field={speaker.name} />
+              <Text tag="p" className="speaker_name" field={speaker.name} />
+              <Text tag="p" field={speaker.jobTitle} />
             </div>
-          </a>
-        </Link>
-      ));
+          </div>
+        </a>
+      </Link>
+    ));
 
   return (
-    <div className="item-grid">
-      <div className="grid-content">{speakers}</div>
+    <div className="featured-speakers">
```

Comment:
I see that this `div` was added just to add a new component-specific className, but that `featured-speakers` class does not define any styles.

To keep the HTML as simple as possible, we could add the class to the existing div to add additional styles to the item-grid only for this component:
```jsx
<div className="featured-speakers item-grid">
  <div className="grid-content">{speakers}</div>
</div>
```
```css
.featured-speaker.item-grid {
  .grid-content {
    @apply ...;
    ...
  }
}
```